code/modules/power/cell.dm

```diff
@@ -10,6 +10,8 @@
 	throw_speed = 2
 	throw_range = 5
 	w_class = 2
+	var/part_type = "cell" //I blame vehicles
+	var/weight = 0 //VEHICLES
```

Comment:
Also not actually being used. These redundant variable seem to be caused by you trying to convert this object to another. Possibly create a dedicated cell type rather than adding variables that aren't used, if this is the issue?